oteapi_optimade/dlite/entities/OPTIMADEStructureAttributes.yaml

```
@@ -42,7 +42,7 @@ properties:
     type: float
     shape: [dimensionality, dimensionality]
     unit: Ångström
-    description: The three lattice vectors in Cartesian coordinates, in ångström (Å).
+    description: The three lattice vectors in Cartesian coordinates, in Ångström (Å).
```

This description is lifted (and meant to be equal to) the OPTIMADE specification: https://github.com/Materials-Consortia/OPTIMADE/blob/develop/optimade.rst#lattice-vectors
```suggestion
    description: The three lattice vectors in Cartesian coordinates, in ångström (Å).
```